frontend/SiraNoMap.html

```diff
@@ -10,12 +10,17 @@
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
 <!--        <link rel="stylesheet" href="assets/application/conoscenze_ambientali/css/skin-mappa.css">-->
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="https://cesiumjs.org/releases/1.17/Build/Cesium/Widgets/widgets.css" />
```

Comment:
I don't think we should include Cesium related css and js, since we are not including the 3D view, could you revert these changes, please?

Reply:
Sure